packages/frameworks/src/frameworks.ts

```diff
@@ -0,0 +1,1605 @@
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `react-scripts build`',
+      },
+      devCommand: {
+        value: 'react-scripts start',
+      },
+      outputDirectory: {
+        value: 'build',
+      },
+    },
+    dependency: '@ionic/react',
+    devCommand: 'react-scripts start',
+    buildCommand: 'react-scripts build',
+    getOutputDirName: async () => 'build',
+    defaultRoutes: [
+      {
+        src: '/static/(.*)',
+        headers: { 'cache-control': 's-maxage=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src: '/service-worker.js',
+        headers: { 'cache-control': 's-maxage=0' },
+        continue: true,
+      },
+      {
+        src: '/sockjs-node/(.*)',
+        dest: '/sockjs-node/$1',
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        headers: { 'cache-control': 's-maxage=0' },
+        dest: '/index.html',
+      },
+    ],
+  },
+  {
+    name: 'Create React App',
+    slug: 'create-react-app',
+    demo: 'https://react-functions.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/react.svg',
+    tagline: 'Create React App allows you to get going with React in no time.',
+    description:
+      'A React app, bootstrapped with create-react-app, and a Serverless Function API.',
+    website: 'https://create-react-app.dev',
+    detectors: {
+      some: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"react-scripts":\\s*".+?"[^}]*}',
+        },
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"react-dev-utils":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `react-scripts build`',
+      },
+      devCommand: {
+        value: 'react-scripts start',
+      },
+      outputDirectory: {
+        value: 'build',
+      },
+    },
+    dependency: 'react-scripts',
+    devCommand: 'react-scripts start',
+    buildCommand: 'react-scripts build',
+    getOutputDirName: async () => 'build',
+    defaultRoutes: [
+      {
+        src: '/static/(.*)',
+        headers: { 'cache-control': 's-maxage=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src: '/service-worker.js',
+        headers: { 'cache-control': 's-maxage=0' },
+        continue: true,
+      },
+      {
+        src: '/sockjs-node/(.*)',
+        dest: '/sockjs-node/$1',
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        headers: { 'cache-control': 's-maxage=0' },
+        dest: '/index.html',
+      },
+    ],
+  },
+  {
+    name: 'Create React App (ejected)',
+    slug: 'create-react-app',
```

Comment:
Why are there two entries for `create-react-app`? I think you can remove this one.